### PR TITLE
Chore: Remove console output from some jest tests

### LINKED ISSUE #45025: Chore: Clean up logs and warnings from BI unit tests
The following tests are outputting errant logs and warnings to the console during unit tests. Some of these are pointing out genuine errors and things that should be looked at, while others might be a bit more benign but should be cleaned up to make testing more pleasant.

In the future we may consider making console logs to be a failure condition in tests, so it would be good to sort these out sooner 😇

 - [ ] 1 log  from `packages/grafana-ui/src/components/GraphNG/nullInsertThreshold.test.ts`

See [this comment](https://github.com/grafana/grafana/issues/42359#issuecomment-1032179575) for common warnings and how to fix them.

If you have any questions be sure to reach out to the User Essentials team ✌🏻 

Part of https://github.com/grafana/grafana/issues/42359

### PR DESCRIPTION
**Pull request by ashharrison90**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes https://github.com/grafana/grafana/issues/45025
For https://github.com/grafana/grafana/issues/45028

**Special notes for your reviewer**:

### CODE REVIEW

File: public/app/core/utils/explore.test.ts [hunk 1]
```diff
@@ -113,31 +113,6 @@ describe('state functions', () => {
...
-          },
-        ],
-        range: {
-          from: 'now-5h',
-          to: 'now',
-        },
-      };
-      expect(serializeStateToUrlParam(state, true)).toBe(
-        '{"datasource":"foo","queries":[{"expr":"metric{test=\\"a/b\\"}","refId":"A"},{"expr":"super{foo=\\"x/z\\"}","refId":"B"}],"range":{"from":"now-5h","to":"now"}}'
-      );
```
Comment (ashharrison90) [inline public/app/core/utils/explore.test.ts:139]: as far as i can tell, these don't test anything that the other tests aren't already testing so i think they can just be removed 😄

Comment (gelicia) [reply public/app/core/utils/explore.test.ts:139]: @Elfo404 This is what you were saying about testing the deprecated `compact` parameter - right?

Comment (Elfo404) [reply public/app/core/utils/explore.test.ts:139]: yeah, basically my reasoning for not removing them back then was to make sure the `compact` parameter was not going to assume a different meaning before we actually removed it. but honestly i agree with @ashharrison90 they are practically useless

**Review by yaelleC — APPROVED** [review]
Approving for the datasource file 👍  LGTM!

Will let @Elfo404 (or someone else with more context than me) decide about the deleted `explore` tests whether it's best to remove now or wait until 9.0 to be sure about the `compact` parameter.

**Review by mckn — APPROVED** [review]
LGTM!

**Review by Elfo404 — APPROVED** [review]
LGTM 🚀

**Pull request merged**